src/hotspot/share/runtime/vmStructs.cpp

```diff
@@ -1261,6 +1263,7 @@
   declare_toplevel_type(FileMapHeader)                                    \
   declare_toplevel_type(CDSFileMapRegion)                                 \
   declare_toplevel_type(UpcallStub::FrameData)                            \
+  declare_toplevel_type(UnloadableMethodHandle)                           \
```

Comment:
So are these left for the async profiler?

Reply:
Yes, see https://github.com/async-profiler/async-profiler/issues/1260 that is filed already.